spec/rails_helper.rb

```diff
@@ -7,6 +7,13 @@
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 
+require 'simplecov'
+SimpleCov.start
+if ENV['CI']=='true'
+  require 'codecov'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.